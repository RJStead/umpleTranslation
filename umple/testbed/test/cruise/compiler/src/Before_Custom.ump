generate Java "../src-gen-umple";

class BeforeCustom
{
    before custom foo {
        int a = 1;
    }
    
    public void foo() {
        b = 0;
        a++;
    }
}
