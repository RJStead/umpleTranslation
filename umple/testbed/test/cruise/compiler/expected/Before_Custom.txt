Before_Custom.ump:10: error: cannot find symbol
    b = 0;
    ^
  symbol:   variable b
  location: class BeforeCustom
1 error